lib/block/blockheader.js

```diff
@@ -212,6 +217,22 @@ BlockHeader.prototype.getTargetDifficulty = function getTargetDifficulty(info) {
 };
 
 /**
+ * @link https://en.bitcoin.it/wiki/Difficulty
+ * @param {Number} genesisBits
```

Comment:
Also, the test case for passing in the bits isn't covered. I don't currently see a need for the argument yet, so another option would be to remove the argument, and we can add if needed later.